tasks/CMakeLists.txt

```diff
@@ -10,7 +10,8 @@ add_dependencies(${CONTROLDEV_TASKLIB_NAME}
 
 TARGET_LINK_LIBRARIES(${CONTROLDEV_TASKLIB_NAME}
     ${OrocosRTT_LIBRARIES}
-    ${CONTROLDEV_TASKLIB_DEPENDENT_LIBRARIES})
+    ${CONTROLDEV_TASKLIB_DEPENDENT_LIBRARIES}
+    /home/henrique/dev-tidewise/install/seasocks/lib/libseasocks.so)
```

Comment:
hardcoded full path to your machine .... hum

Reply:
After setting the package set and all I will set that.